tests/pageobject/bayutPropertyPageSearchFilters.page.js

```diff
@@ -0,0 +1,201 @@
+import Page from "./page.js"
+
+class xPathFilters extends Page{
+    //element, type object, methods getText, etc
+    /**
+     * out: value: 
+        { ELEMENT: '0.6471768287987443-1',
+            'element-6066-11e4-a52e-4f735466cecf': '0.6471768287987443-1' },
+            selector: '//a[contains(@title, "Bayut.com")]',
+            _status: 0 
+        } */
+
+    get eBayutLogo(){
+        return $(`//*[contains(@title, "Bayut.com")]`)
+    }
+
+
+    // selector type string
+    /**
+     * out: //a[contains(@title, "Bayut.com")
+     */
+    get selectorBayutLogo(){
+        return $(`//a[contains(@title, "Bayut.com")]`).selector
+    }
+
+    //element type function
+    /**
+     * out; function () { [native code] }
+     */
+    get elementFunctionBayutLogo(){
+        return $(`//a[contains(@title, "Bayut.com")]`).element
+    }
+
+//-------------------------------------------------------------------------------------------------
+
+    // get the buy btn
+    get ebuyBtn(){
+        return $(`${this.eBayutLogo.selector}/following-sibling::div/span[1]`)
```

Comment:
Could also be done like this.

```suggestion
        return $('//*[@aria-label="Mini search form"]//*[text()="For Sale"]')
```

This way, we have less dependability on other elements. We don't have to rely on `eBayutLogo` or the hierarchy in the DOM

Reply:
The element that you suggested doesn't exist. Instead i have change it with $('//*[text() = "Buy"]')